src/mediapipe_calculators/modelapiovmsadapter.cc

```diff
@@ -172,6 +175,9 @@ void OVMSInferenceAdapter::loadModel(const std::shared_ptr<const ov::Model>& mod
         ASSERT_CAPI_STATUS_NULL(OVMS_ServableMetadataGetOutput(servableMetadata, id, &tensorName, &datatype, &dimCount, &shapeMin, &shapeMax));
         outputNames.emplace_back(tensorName);
     }
+    const ov::AnyMap* servableMetadataRtInfo;
+    ASSERT_CAPI_STATUS_NULL(OVMS_ServableMetadataGetInfo(servableMetadata, const_cast<void**>(reinterpret_cast<const void**>(&servableMetadataRtInfo))));
+    this->modelConfig = *servableMetadataRtInfo;
```

Comment:
```suggestion
    ov::AnyMap* servableMetadataRtInfo;
    ASSERT_CAPI_STATUS_NULL(OVMS_ServableMetadataGetInfo(servableMetadata, reinterpret_cast<void**>(&servableMetadataRtInfo)));
    this->modelConfig = *servableMetadataRtInfo;
```

Why those const casts?